packages/story-editor/src/components/panels/document/taxonomies/taxonomies.js

```diff
@@ -18,18 +18,146 @@
  * External dependencies
  */
 import { __ } from '@web-stories-wp/i18n';
+import {
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+  BUTTON_VARIANTS,
+  DropDown,
+  Headline,
+  Input,
+  Text,
+  themeHelpers,
+  THEME_CONSTANTS,
+} from '@web-stories-wp/design-system';
+import { useCallback, useMemo, useState } from '@web-stories-wp/react';
+import { v4 as uuidv4 } from 'uuid';
 /**
  * Internal dependencies
  */
+import styled from 'styled-components';
 import { SimplePanel } from '../../panel';
+import { HierarchicalInput } from '../../../form';
+import { noop } from '../../../../utils/noop';
+
+const ContentArea = styled.div`
+  label,
+  * > label {
+    ${({ theme }) =>
+      themeHelpers.expandPresetStyles({
+        preset:
+          theme.typography.presets.label[
+            THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL
+          ],
+        theme,
+      })};
+
+    color: ${({ theme }) => theme.colors.fg.secondary};
+  }
+`;
+
+const ContentHeading = styled(Headline).attrs({
+  as: 'h3',
+  size: THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.XXX_SMALL,
+})`
+  margin: 4px 0 16px;
+  font-weight: ${({ theme }) => theme.typography.weight.regular};
+`;
+
+const AddNewCategoryArea = styled.div`
+  margin: 24px 0 16px;
+`;
+
+const LinkButton = styled(Button).attrs({
+  variant: BUTTON_VARIANTS.LINK,
+})`
+  margin-bottom: 16px;
+
+  ${({ theme }) =>
+    themeHelpers.expandPresetStyles({
+      preset:
+        theme.typography.presets.link[
+          THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL
+        ],
+      theme,
+    })};
+
+  font-weight: 500;
+`;
+
+const Label = styled(Text).attrs({
+  forwardedAs: 'label',
+  size: THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL,
+})`
+  display: inline-block;
+  margin: 12px 0;
+`;
+
+const AddNewCategoryButton = styled(Button).attrs({
+  variant: BUTTON_VARIANTS.RECTANGLE,
+  size: BUTTON_SIZES.SMALL,
+  type: BUTTON_TYPES.SECONDARY,
+})`
+  margin-top: 20px;
+`;
 
 function TaxonomiesPanel({ ...props }) {
+  const [showAddNewCategory, setShowAddNewCategory] = useState(false);
+  const [newCategoryName, setNewCategoryName] = useState('');
+  const dropdownId = useMemo(uuidv4, []);
+
+  const handleShowAddNewCategoryClick = useCallback(() => {
+    setShowAddNewCategory(true);
+  }, []);
+
+  const handleChangeNewCategoryName = useCallback((evt) => {
+    setNewCategoryName(evt.target.value);
+  }, []);
+
+  const handleAddCategory = useCallback(() => {
+    setShowAddNewCategory(false);
+  }, []);
+
   return (
     <SimplePanel
       name="taxonomies"
       title={__('Categories and Tags', 'web-stories')}
       {...props}
-    />
+    >
+      <ContentArea>
+        <ContentHeading>{__('Categories', 'web-stories')}</ContentHeading>
```

Comment:
awesome. thanks! will incorporate this into the tags panel when that gets merged.